social/email/61-email.js

```diff
@@ -352,7 +352,10 @@ module.exports = function(RED) {
                         return;
                     }
                     else {
-                        imap.search([ node.criteria ], function(err, results) {
+                        var criteria = ((node.criteria === '_msg_')?
+                                        (msg.criteria || []):
```

Comment:
should the default be ALL ?

Reply:
It could be the same default as the new node.
And it is still possible to pass `[]` anyhow.

```suggestion
                                        (msg.criteria || ["UNSEEN"]):
```

Reply:
yes probably better if it does the default if not set.

Reply:
Applied.
I did test locally.